graders/c/requirements.txt

```diff
@@ -1,4 +1,4 @@
-matplotlib==3.7.2
-numpy==1.25.2
+matplotlib==3.8.0
+numpy==1.26.0
```

Comment:
For my own knowledge: why *does* the C grader include these, or indeed any additional Python packages at all?

Reply:
These are used by some of the questions that originally used this grader before it was made official, and are kept for compatibility reasons (instead of forcing those questions to create custom images with these packages). Some questions use `matplotlib` to generate images with the result, or `numpy` to compute the answer for a random set of inputs.